asetalias/Logistics-and-Event-Contributions
# chore: Add an issue form for event proposals

## What
Issue form for event proposals; links to our Discord, Telegram and membership form in the issue template chooser.

### Screenshots

- Template chooser
  ![image](https://user-images.githubusercontent.com/72784348/201467665-9965fbd8-7191-4026-81eb-aac04dad21ec.png)

- Proposal form
  ![image](https://user-images.githubusercontent.com/72784348/201467709-f00fd291-a127-4e9f-b980-42750c7a1b8f.png)

### FAQ

> How do I test this?

Try making a new issue on my fork https://github.com/ditsuke/logistics-and-event-contributions. Since my `master` branch contains the changes from this PR, you'll be able to see the selector and issue form.
